MAINTAINERS.md

```diff
@@ -0,0 +1,145 @@
+# MCP Maintainers
+
+This document lists current maintainers and working groups in the Model Context Protocol project.
+
+**Last updated:** July 23, 2025
+
+## Lead Maintainers
+
+| Name                                                     |
+| -------------------------------------------------------- |
+| [Justin Spahr-Summers](https://github.com/jspahrsummers) |
+| [David Soria Parra](https://github.com/dsp-ant)          |
```

Comment:
Now that the table has only one column, it might be easier to read as a simple list.

```suggestion
- Justin Spahr-Summers](https://github.com/jspahrsummers)
- David Soria Parra](https://github.com/dsp-ant)
```